### PR TITLE
incus/storage_volume: Add yaml support for create

### PR DESCRIPTION
**Pull request by awalvie**
Part of lxc#741

### CODE REVIEW

File: cmd/incus/storage_volume.go [hunk 1]
```diff
@@ -612,15 +616,29 @@ func (c *cmdStorageVolumeCreate) Run(cmd *cobra.Command, args []string) error {
...
-	vol.Name = volName
-	vol.Type = volType
-	vol.ContentType = c.flagContentType
-	vol.Config = map[string]string{}
+	vol := api.StorageVolumesPost{
+		Name:             volName,
+		Type:             volType,
+		ContentType:      c.flagContentType,
+		StorageVolumePut: volumePut,
+	}
```
Comment (stgraber) [inline cmd/incus/storage_volume.go:641]: This causes build failures as you are no longer initializing `Config` to an empty map.

**Comment by stgraber** [comment]
Needs a tiny fix and a rebase.

**Comment by awalvie** [comment]
Pushed the fixes!

**Comment by awalvie** [comment]
I don't know why it closed, sorry about that.

**Comment by awalvie** [comment]
Couple of stupid mistakes, sorry about the force pushes.

**Pull request merged**